### DISCUSSION TITLE
安装失败

### DISCUSSION THREAD

**Discussion by CrazyWuzj** [fraxiinus/ReplayBook#267] [Troubleshooting]
![image](https://user-images.githubusercontent.com/132224297/235370047-fcc24a45-84b4-4a60-b3c6-fed9bd3b192e.png)
![image](https://user-images.githubusercontent.com/132224297/235370095-5c958cf2-9c60-4f0b-9095-0b2fa1e019e8.png)

**Comment by fraxiinus** [comment]
Please find and attach the the newest log file in this folder: 
![explorer_IwbInf1PGe](https://user-images.githubusercontent.com/8365233/235373764-11ae57c0-270e-47b2-a979-e86e5a60a477.png)


![explorer_CwC5PP5nAo](https://user-images.githubusercontent.com/8365233/235373765-7a1a8e3a-8b00-45c8-bc4a-e269c59af40a.png)

**Comment by fraxiinus** [comment]
New version should fix this:

https://github.com/fraxiinus/ReplayBook/releases/tag/1.7.1
